src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -1046,15 +1047,15 @@ async Task ResolveEntityRef(Address destinationAddress, ActorPath entityPath)
                         // NOTE: in the event that we're querying a shard's location from a ShardRegionProxy
                         // the shard may not be technically "homed" inside the proxy, but it does exist.
 #pragma warning disable CS4014
-                        ResolveEntityRef(GetNodeAddress(shardRegionRef), shardRegionRef.Path / shardId / shardId); // needs to run as a detached task
+                        ResolveEntityRef(GetNodeAddress(shardRegionRef), shardRegionRef.Path / shardId / entityId); // needs to run as a detached task
```

Comment:
The actor path should be `shardRegionRef.Path / shardId / entityId` not `shardRegionRef.Path / shardId / shardId`

Reply:
Yep, my mistake